docs/source/API/core/view/create_mirror.rst

```diff
@@ -15,14 +15,22 @@ A common desired use case is to have a memory allocation in GPU memory and an id
 Usage
 -----
 
+The key difference between ``create_mirror`` and ``create_mirror_view`` is the following: ``create_mirror`` `always` allocates new memory in the specified space (shown below for host space), while ``create_mirror_view`` only allocates memory if the View to be mirrored (``a_view``) is not already accessible from the specified space, and otherwise simply returns ``a_view``.
+Use ``create_mirror_view`` when you want to skip the memory allocation and copy within the same execution space, and use ``create_mirror`` if you need the data to be independent, e.g. for overlapping computation and I/O.
```

Comment:
In my opinion, this fits better in the block above. Also, the last sentence seems redundant, in particular, since the interface is primarily about memory spaces and not execution spaces.

Reply:
Feel free to move it around. I think an illustrative example (the last sentence) is very helpful, so some form of it should remain in my opinion.
For execution vs memory space: The example under "Usage" already had a parameter called "ExecSpace", so I just stuck to that. But yeah, I wasn't sure about that part.

Reply:
In the sentence
"Use ``create_mirror_view`` when you want to skip the memory allocation and copy within the same execution space, and use ``create_mirror`` if you need the data to be independent, e.g. for overlapping computation and I/O."
we could remove "execution". That would now also fit with the example code.

Reply:
Hmm ... stating that a copy is avoided implies, that a copy is always required.

Reply:
I would drop the sentence with the examples. It sounds too much like a recommendation tbh.

Reply:
That sentence was a bit tricky. But while it's not airtight, I think it gets the message across with good brevity and clarity.
It is written like a recommendation, and I think that's exactly what is needed when functions with this level of similarity are provided. But we could also write something like
"Using ``create_mirror_view`` allows skipping the memory allocation and potential copies within the same execution space, while ``create_mirror`` can be used if you need the data to be independent, e.g. for overlapping computation and I/O."

Feel free to make a different suggestion. But leaving out the example sounds like the worst option to me.